src/VisualStudio/Core/Impl/ProjectSystem/CPS/CPSProject_IWorkspaceProjectContext.cs

```diff
@@ -181,6 +173,32 @@ public void SetProperty(string name, string? value)
             {
                 _visualStudioProject.DependencyNodeTargetIdentifier = value;
             }
+            else if (name == AdditionalPropertyNames.TargetRefPath)
+            {
+                // If we don't have a path, always set it to null
+                if (string.IsNullOrEmpty(value))
+                {
+                    _visualStudioProject.OutputRefFilePath = null;
+                }
+                else
+                {
+                    // If we only have a non-rooted path, make it full. This is apparently working around cases
+                    // where CPS pushes us a temporary path when they're loading. It's possible this hack
```

Comment:
Temporary path?

Reply:
Copied from the other output fine handling